File: Python/palindrome.py
number=input("Enter the Number: ")
n=len(number)
left=0
right=n-1
count=0
while left<right:
    if number[left]==number[right]:
        count+=1
    left+=1
    right-=1
if count==n//2:
    print("Entered Number is Palindrome")
else:
    print("Entered number is not a palindrome")

File: Java/Palindrome.java
package Daywise;

public class Palindrome {
    public static void main(String[] args){
        String number = "4554";
        int count=0;
        int left=0;
        int n = number.length();
        int right=n-1;
        while(left<right){
            if(number.charAt(left)==number.charAt(right)){
                count=count+1;
            }
            left=left+1;
            right=right-1;
        }
        if(number.length()/2==count){
            System.out.print("Is a palindrome");
        }else {
            System.out.println("Is not a palindrome");
        }
    }
}
